workspaces/src/network/variants.rs

```diff
@@ -90,15 +112,85 @@ where
     /// }
     /// ```
     ///
-    pub async fn dev_create_account(&self) -> Result<Account> {
-        let (id, sk) = self.dev_generate().await;
+    pub async fn dev_create_tla(&self) -> Result<Account> {
+        let (id, sk) = self.generate_tla_credentials().await;
         let account = self.create_tla(id.clone(), sk).await?;
         Ok(account.into_result()?)
     }
 
+    pub async fn dev_deploy_tla(&self, wasm: &[u8]) -> Result<Contract> {
+        let (id, sk) = self.generate_tla_credentials().await;
+        let contract = self.create_tla_and_deploy(id.clone(), sk, wasm).await?;
+        Ok(contract.into_result()?)
+    }
+}
+
+impl<T> Worker<T>
+where
+    T: DevNetwork + 'static,
+{
+    pub async fn create_sponsored_account(
+        &self,
+        subaccount_prefix: AccountId,
+        sk: SecretKey,
+    ) -> Result<Execution<Account>> {
+        if subaccount_prefix.as_str().contains('.') {
+            return Err(crate::error::ErrorKind::Io
+                .custom("Subaccount prefix for sponsored account cannot contain '.'"));
+        }
+        let res = self
+            .workspace
+            .create_sponsored_account(self.clone().coerce(), subaccount_prefix, sk)
+            .await?;
+
+        for callback in self.tx_callbacks.iter() {
+            callback(res.details.total_gas_burnt)?;
+        }
+
+        Ok(res)
+    }
+
+    pub async fn create_sponsored_account_and_deploy(
+        &self,
+        subaccount_prefix: AccountId,
+        sk: SecretKey,
+        wasm: &[u8],
+    ) -> Result<Execution<Contract>> {
+        if subaccount_prefix.as_str().contains('.') {
+            return Err(crate::error::ErrorKind::Io
+                .custom("Subaccount prefix for sponsored account cannot contain '.'"));
+        }
+        let res = self
+            .workspace
+            .create_sponsored_account_and_deploy(self.clone().coerce(), subaccount_prefix, sk, wasm)
+            .await?;
+
+        for callback in self.tx_callbacks.iter() {
+            callback(res.details.total_gas_burnt)?;
+        }
+
+        Ok(res)
+    }
+
+    pub async fn dev_generate(&self) -> (AccountId, SecretKey) {
```

Comment:
`dev_generate_credentials`
would be easier to find

Reply:
Guess we can change it to `generate_sponsored_credentials` since we use it to create sponsored accounts. WDYT?